app/controllers/component_items_controller.rb

```diff
@@ -36,7 +36,7 @@ def items_params
   end
 
   def validate_items
-    MetadataPresenter::ValidateSchema.validate(params[:data], 'definition.select')
+    MetadataPresenter::ValidateSchema.validate(params[:data].to_json, 'definition.select')
```

Comment:
if this was passing without using `to_json`, i.e autocomplete items were being successfully saved to the metadata api previously, then i am not sure it is required now. we haven't changed how the payload is sent from the editor have we?